_config.yml

```diff
@@ -1,6 +1,6 @@
 # Site settings
-title: openHAB
-description: Open Source Smart Home Solution
+title: openHAB 2.0
```

Comment:
I would suggestion to only talk about openHAB 2 everywhere from now on. The closer we come to an 2.0 release, the sooner 2.1 will be on the horizon and we won't want to update everything again then!